include/grpcpp/impl/server_builder_plugin.h

```diff
@@ -44,10 +47,10 @@ class ServerBuilderPlugin {
 
   /// InitServer will be called in ServerBuilder::BuildAndStart(), after the
   /// Server instance is created.
-  virtual void InitServer(ServerInitializer* si) = 0;
+  virtual void InitServer(grpc_impl::ServerInitializer* si) = 0;
```

Comment:
I'm thinking this may create confusions for the end users to have the namespace in the public arguments like this, but this is going to be a recurring issue anyway. We can try polishing this later.

Reply:
Filed - https://github.com/grpc/grpc/issues/18573